src/optimizer/query_planner.c

```diff
@@ -1128,7 +1128,7 @@ qo_plan_print_sort_spec_helper (PT_NODE * list, bool is_iscan_asc, FILE * f, int
 
       if (TP_TYPE_HAS_COLLATION (TP_DOMAIN_TYPE (list->info.sort_spec.pos_descr.dom))
 	  && TP_DOMAIN_COLLATION (list->info.sort_spec.pos_descr.dom) != LANG_SYS_COLLATION
-	  && TP_DOMAIN_COLLATION (list->info.sort_spec.pos_descr.dom) != (LANG_SYS_COLLATION + COLL_TS)
+	  && TP_DOMAIN_COLLATION (list->info.sort_spec.pos_descr.dom) != (LANG_SYS_COLLATION + COLL_TI)
```

Comment:
How about deleting this condition?
I know this condition is for the unusual 'char' case of the previous modification.

Reply:
the user can specify ti-collation for char type, so this condition is needed.

Reply:
I fixed it, this code is removed.